envisage/ui/tasks/preferences_category.py

```diff
@@ -18,7 +18,7 @@ class PreferencesCategory(HasTraits):
     id = Str
 
     # The user-visible name of the category.
-    name = Unicode
+    name = Str
```

Comment:
My fingers were sorely tempted to add `()` to the end of this line, but we can do all that in a separate PR.